app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parsed_command = command.split()
+    path = parsed_command[-1].split("/")
+    directory = ""
+
+    for i in range(len(path) - 1):
+        directory += path[i]
+        if not os.path.exists(directory):
+            os.mkdir(directory)
+        directory += "/"
```

Comment:
use `os.path.join`  to build correct path for a file

Reply:
I am not using os.path.join in next commit, because i already have got correct directory format with os.path.split(), but i understand your hint. Thank you.